### PR TITLE
TE-448 Use dynamic web port: Use osgi port for web view

### PR DESCRIPTION
**Pull request by kpanier**
Using OSGi Web port vairable to access the test run view and rest services. This avoids hard coded ports.

### CODE REVIEW

File: rcp/org.testeditor.rcp4.views.tcltestrun/src/org/testeditor/rcp4/views/tcltestrun/console/TestExecutionLogViewPart.xtend [hunk 1]
```diff
@@ -26,7 +26,9 @@ class TestExecutionLogViewPart {
 	@PostConstruct
 	def void buildUI(Composite parent) {
 		browser = new Browser(parent, SWT.NORMAL)
-		browser.setUrl("http://localhost:19091/testlogs/web-app/index.html")
+		val url = "http://localhost:" + System.getProperty("org.osgi.service.http.port") +
```
Comment (franzbecker) [inline rcp/org.testeditor.rcp4.views.tcltestrun/src/org/testeditor/rcp4/views/tcltestrun/console/TestExecutionLogViewPart.xtend]: Should we maybe allow to set a specific port as well?
We could check a Test-Editor system property first and if it's not set use the org.osgi.service.http.port.
Also I'd prefer assigning the port to a local variable and use the `val url = '''http://localhost:«port»/testlogs/web-app/index.html'''` syntax.

Comment (kpanier) [inline rcp/org.testeditor.rcp4.views.tcltestrun/src/org/testeditor/rcp4/views/tcltestrun/console/TestExecutionLogViewPart.xtend]: I used the osgi Property to Reuse the existing osgi Server. If we have a custum Port we should also manage a separate Server instance. 
For rcp there is in another branch an aproach to select a Free Port. For an Developer in the ide i prefer to let him manage it with thr Default Property.
Further work could be a config to a Remote Server.

I will fix the sugggested String building Way

Comment (franzbecker) [inline rcp/org.testeditor.rcp4.views.tcltestrun/src/org/testeditor/rcp4/views/tcltestrun/console/TestExecutionLogViewPart.xtend]: :+1:

File: rcp/org.testeditor.web.testexecutionlogview/app/testLog.service.ts [hunk 2]
```diff
@@ -22,9 +22,9 @@ export class TestLogService {
         ).catch(this.handleError);
     }
 
-    getTestExecutionLogGroups(testExecLog: TestExecutionLog): Promise<LogGroup[]> {
+    getTestExecutionLogGroups(testExecLog: TestExecutionLog): Promise<TestExecutionLog> {
```
Comment (gunther-bachmann) [inline rcp/org.testeditor.web.testexecutionlogview/app/testLog.service.ts]: Should the function be renamed to getTestExecutionLog ?

**Pull request merged**